WalletWasabi.Fluent/Helpers/StartupHelper.cs

```diff
@@ -48,5 +49,27 @@ private static void ModifyRegistry(bool runOnSystemStartup, string pathToExeFile
 				key.DeleteValue(nameof(WalletWasabi), false);
 			}
 		}
+
+		private static void ModifyMacOsLoginItems(bool runOnSystemStartup)
+		{
+			if (!RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+			{
+				throw new InvalidOperationException("Running osascript can only be done on MacOS.");
```

Comment:
I believe the correct way to write the name is:

```suggestion
				throw new InvalidOperationException("Running osascript can only be done on macOS.");
```
(https://en.wikipedia.org/wiki/MacOS)

Reply:
Right.